src/password/landing.js

```diff
@@ -1,17 +1,22 @@
 document.addEventListener('DOMContentLoaded', () => {
 if(localStorage.getItem('storedHashedPassword')){
+    document.getElementById('welcome-button').style.display = 'none'
 //Wait for input password to be submitted
 document.getElementById('input-password-form').addEventListener('submit', async function(event){
+    console.log("here")
     event.preventDefault();
     const inputPassword = document.getElementById('password').value;
```

Comment:
On the input, add a class "js-password" or something along those lines, and switch to using `querySelector(".js-password")`. 
We're avoiding using ids and normal classes and jsut having "js-" classes to select elements in JS so that if the class/id names change as css and html change, the js functionality remains untouched.